src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  margin: auto;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 7px;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-black);
+  margin-top: 40px;
+  width: 100%;
+}
+
+.card__text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__text--code {
+  color: var(--color-grey);
+  margin-top: 4px;
+  align-self: flex-start;
+}
+
+.card__text--review {
+  color: var(--color-black);
+}
+
+.card__container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__container--rate {
+  margin-top: 16px;
+}
+
+.card__container--price {
+  margin-top: 24px;
+}
+
+.card__text--price {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-grey);
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-black);
+  text-align: right;
+}
+
+.card__link {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--color-white);
+  text-decoration: none;
+  background: var(--color-link);
+  border-radius: 5px;
+  margin-top: 16px;
+  width: 100%;
+  border: 1px solid var(--color-link);
+  padding: 12px 0;
```

Comment:
just add fixed height

Reply:
Initially I have fixed height for this element.  Your Colleague have checked and advised : [Viktor-Kost](https://github.com/Viktor-Kost) [4 days ago](https://github.com/mate-academy/layout_product-cards/pull/2362#discussion_r1056534363)
It i better not hardcode height for elements with text. Use just padding.  -->   I have changed. I guess there is no one point of view. May I stay padding?

Reply:
It's ok